### ISSUE TITLE
idr MD5 mismatch

### ISSUE THREAD

**Issue by nsoranzo** [bioconda/bioconda-recipes#1491]
From `recipes/idr/meta.yaml`:
```yaml
source:
  fn: 2.0.3.tar.gz
  url: https://github.com/nboley/idr/archive/2.0.3.tar.gz
  md5: 14219095e41650221634e6097842568d
```
but:
```bash
$ wget https://github.com/nboley/idr/archive/2.0.3.tar.gz
$ md5sum 2.0.3.tar.gz 
5873427fd7c5bd474755e601df60e864  2.0.3.tar.gz
```

This is breaking Travis: https://travis-ci.org/bioconda/bioconda-recipes/jobs/128365991

Ping @nboley.

**Comment by daler** [comment]
This had passed the PR test, @nboley was a different version with the same filename uploaded between then and now?

To avoid holding everything else up I'm going to temporarily disable the IDR build.

**Comment by daler** [comment]
Note we're having other build issues as well (xref #1487). Pretty sure it's unrelated, but you never know . . .

**Comment by nboley** [comment]
I submitted a PR that fixes this.

**Comment by daler** [comment]
Yep, first PR was #1481 that bumped to 2.0.3; that passed. The second PR was #1485 that changed the filename to match that of the downloaded URL, that also passed and was merged. 

However, upon merging #1485, when travis-ci tried to build it (the very same version that just passed!) it got an md5 mistmatch when downloading the file.

As @nsoranzo mentions, the yaml currently says this:

```yaml
source:
  fn: 2.0.3.tar.gz
  url: https://github.com/nboley/idr/archive/2.0.3.tar.gz
  md5: 14219095e41650221634e6097842568d
```

and he reports getting:

```
$ wget https://github.com/nboley/idr/archive/2.0.3.tar.gz
$ md5sum 2.0.3.tar.gz 
5873427fd7c5bd474755e601df60e864  2.0.3.tar.gz
```

And I just tried it and now I get:

```
$ wget https://github.com/nboley/idr/archive/2.0.3.tar.gz
$ md5sum 2.0.3.tar.gz
1bec61b526bb1dcf92a5ab99c27852d0  2.0.3.tar.gz
```

So it appears that the archive release on github is changing. I've never seen this before. Before trying all sorts of things to troubleshoot if this is a general problem with github, travis, or our build system, do you know of any changes that might have happened to the file at https://github.com/nboley/idr/archive/2.0.3.tar.gz over the past day or so?

**Comment by nboley** [comment]
This is because I accidentally re-tagged a new patch as 2.0.3 rather than 2.0.4 (as I should have). 

This is completely about me being stupid - not problem with github, travis, or the build system.

I'm really sorry for all of this confusion.

**Comment by daler** [comment]
Phew! Good to know. Once the file settles down, feel free to submit a PR to revert #1492.

**Comment by nboley** [comment]
Submitted and merged [#1495](https://github.com/bioconda/bioconda-recipes/pull/1495)
